designs/embed_eksd_manifest.md

```diff
@@ -0,0 +1,59 @@
+# Embed EKS-D resources in EKS-A cluster
+
+## Introduction
+
+**Problem:** Currently, the EKS Anywhere controller pulls the EKS Distro manifest from the release URL provided in the versions bundle, which requires access to the internet.
+It would be ideal to have this manifest and its CRDs embedded in the cluster to allow the controller to operate without the requirement to make that external call.
+This will eventually be needed for air-gapped support.
+
+### Goals and Objectives
+
+As an EKS Anywhere user:
+
+* I want the EKSA controller to be able to operate without internet access
+* I want EKSD components to be supported during upgrade
+
+## Overview of Solution
+
+The EKS Distro manifest and the release CRD will be applied directly to the cluster.
+This will allow the controller to grab the EKSD release resources from within the cluster instead of reading in the EKSD manifest from the internet when fetching the applied spec.
+
+### Solution Details
+
+With this feature, the EKSD manifest & its CRD will be applied directly to the cluster.
+We will do this similarly to how we apply the EKSA components to the cluster, introducing a new task to the create workflow called `InstallEksdComponentsTask`.
+We currently don’t pull in the EKSD release CRD in the bundle, so I propose introducing a new field `crdUrl` in the EKSD Bundle to refer to the file in S3 that stores this CRD.
```

Comment:
Actually in practice, this is something that is stored as part of the actual manifest URL that we have linked in the eksD. I would be curious to see what the effort required is to just edit our current manifest files to include the CRDs themselves instead of maintaining a separate URL

Reply:
Looking into this option!